Changelog.md

```diff
@@ -1,9 +1,18 @@
 Changelog
 =========
 
-### 2.1.3 (2018-xx-xx)
-
-* Added Mongolian translation.
```

Comment:
why not releasing that in a 2.x release ?

Reply:
You are right: https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2952